sim/ngspice/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Identify OS
+UNAME_OUT="$(uname -s)"
+case "${UNAME_OUT}" in
+    Linux*)     OS=Linux;;
+    *)          OS="${UNAME_OUT}"
+                echo "Unknown OS: ${OS}"
+                exit;;
+esac
+
+./autogen.sh
+./configure --prefix="${PREFIX}" --disable-debug
+make V=1 -j$CPU_COUNT
```

Comment:
```suggestion
make V=1 -j$(nproc)
```

Reply:
Oh I forgot the `CPU_COUNT` variable is set by conda-build. Great, let's keep it.